### ISSUE TITLE
[feature request] multilanguage support

### ISSUE THREAD

**Issue by nicholascw** [ConstantinCezB/Microflux#9]
Any possibilities of adding multi-language support? I understand the app has few words and may make few obstacles even for users don't speak English at all. However I believe supporting more languages could ultimately up the experience into the next level as it'll provide a more native feel. Also looking for ways to contribute to the translations. thx

**Comment by ConstantinCezarBegu** [comment]
Yeah, this can be added. I can do French and Roumanian since those are the languages. Other languages I might need some help. I am going to work on a new update that will support showing articles by categories (miniflux api has been updates). So I can put this in there also.
